src/chapters/arrays/array-methods.rst

```diff
@@ -0,0 +1,151 @@
+.. _array-methods:
+
+Array Methods
+==============
+
+.. index::
+   single: array; methods
+
+As with strings, JavaScript provides us with useful **methods** for arrays.
+Unlike strings, arrays are *mutable* and can be changed.
```

Comment:
This can be omitted as it is previously stated in the chapter.

Reply:
Repetition is always a good thing.  I would prefer to keep the reminder.

Reply:
You convinced me.  Sentence removed.